### PR TITLE
CIV-13583  payment success/application submitted screen

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-13583
https://tools.hmcts.net/jira/browse/CIV-13675

### Change description ###

payment success/application submitted 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by manish14836 — DISMISSED** [review]

**Review by manish14836 — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Review by manish14836 — DISMISSED** [review]

**Review by kenneth-hmcts — DISMISSED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

File: src/main/services/features/generalApplication/applicationFeePaymentConfirmationContent.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export const getGaPaymentSuccessfulButtonContent = (redirectUrl : string) => {
+  return new PaymentSuccessfulSectionBuilder()
+    .addButton('COMMON.BUTTONS.CLOSE_AND_RETURN_TO_DASHBOARD', redirectUrl)
+    .build();
+};
+
+const isApplicationSubmittedWithoutFee = (applicationType: ApplicationTypeOption, caseProgressionHearing: CaseProgressionHearing) => {
+  const caseProgressionHearingDetails = Object.assign(new CaseProgressionHearing(), caseProgressionHearing);
+  return applicationType === ApplicationTypeOption.ADJOURN_HEARING && (caseProgressionHearingDetails.getDurationOfDaysForHearing() >= 14 || !caseProgressionHearingDetails.hearingDate);
```
Comment (paul-pearson) [inline src/main/services/features/generalApplication/applicationFeePaymentConfirmationContent.ts:49]: Could use a constant for the value 14 days.

**Review by kalachandrasekar1 — APPROVED** [review]

File: src/test/utils/mocks/a11y/mock-case-1645882162449409-general-application-payment-sucessful.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (paul-pearson) [inline src/test/utils/mocks/a11y/mock-case-1645882162449409-general-application-payment-sucessful.html]: Word `sucessful` in filename is missing a letter "c", so I think accessibility test will fail.

**Review by paul-pearson — CHANGES_REQUESTED** [review]

File: src/test/utils/mocks/a11y/mock-case-1645882162449409-general-application-payment-sucessful.html (hunk 2 above)
Comment (jeswanth-hmcts) [reply src/test/utils/mocks/a11y/mock-case-1645882162449409-general-application-payment-sucessful.html]: added c in the filename

File: src/main/services/features/generalApplication/applicationFeePaymentConfirmationContent.ts (hunk 1 above)
Comment (jeswanth-hmcts) [reply src/main/services/features/generalApplication/applicationFeePaymentConfirmationContent.ts]: moved to constant

**Review by paul-pearson — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Pull request merged**